syft/frameworks/torch/tensors/native.py

```diff
@@ -21,17 +22,56 @@ class TorchTensor(AbstractTensor):
     """
 
     def __str__(self) -> str:
```

Comment:
This functionality is in abstract tensor

Reply:
Not exactly, the code is a bit different because here you call `native_`